### DISCUSSION TITLE
Whether promotion Screen refresh rate is 24fps when playing 24fps video

### DISCUSSION THREAD

**Discussion by rainlycoris** [iina/iina#5146] [Q&A] [answered]
Or worse, the screen renders at 60fps or even 30fps.
I haven't found anything useful about apple promotion at all.
Can `video-sync = display-resample` make sure it's correct?

**Comment by low-batt** [comment, marked as answer]
I believe the answer is none of the above, it renders at 120fps.

I am typing this on a MacBook Pro with a ProMotion display, so I too want support for this. It did not make the list for the upcoming release, but it is in the list of features we hope to get to.
